tests/python/network_spammer.py

```diff
@@ -0,0 +1,236 @@
+    tx_results = await send_txs(env.horizon_uri, xdrs, expected_statuses=[200, 500, 504])
+
+    # fetch builder sequence number.
+    #
+    # NOTE this is necessary because not all transactions have been processed
+    # when we reach this stage,specifically unprioritized ones.
+    # thus, it is unknown if the sequence has increased or not
+    all_builders = prioritized_builders + unprioritized_builders
+    sequences = await get_sequences(prioritized_builders[0].horizon.horizon_uri,
+                                    [b.address for b in all_builders])
+
+    # update builder sequence number
+    for i, _ in enumerate(all_builders):
+        b = all_builders[i]
+        b.clear()
+        b.sequence = str(sequences[i])
+
+    # create tx result object for each tx for reviewing later on
+    results = []
+    for (hsh, metadata), result in zip(tx_metadata.items(), tx_results):
+        try:
+            ledger = result['ledger']
+        except KeyError:  # probably 504
+            ledger = None
+        results.append({'hash': hsh,
+                        'round': metadata['round'],
+                        'prioritized': metadata['prioritized'],
+                        'ledger': ledger,
+                        'submission_time': submission_time})
+
+    return results
+
+
+LEDGERS = {}
+
+
+def ledger_time(ledger, horizon):
+    """Return ledger close time."""
+    # failed txs have ledger = None
+    # so we return an empty timestamp instead
+    if not ledger:
+        return ''
+
+    global LEDGERS
+    if ledger not in LEDGERS:
+        logging.debug('fetching ledger %d', ledger)
+        date_str = horizon.ledger(ledger)['closed_at']
+        LEDGERS[ledger] = datetime.strptime(date_str, '%Y-%m-%dT%H:%M:%SZ').timestamp()
+    return LEDGERS[ledger]
+
+
+async def main():
+    args = parse_args()
+
+    # init root account (funder) tx builder
+    env = KinEnvironment(NETWORK_NAME, args.horizon[0], args.passphrase)
+    root_builder = Builder(env.name, KinClient(env).horizon, MIN_FEE, root_account_seed(args.passphrase))
+    root_builder.sequence = root_builder.get_sequence()
+
+    # create channel accounts for mass creating accounts later on
+    # these accounts will consume the sequence number for the root builder
+    # so we can parallelize mass account creation
+    account_creator_channels = await create_accounts(env, [root_builder], root_builder.keypair, STARTING_BALANCE, args.horizon)
+
+    # create prioritizer accounts
+    #
+    # NOTE we create as many prioritizers as there are prioritized txs per
+    # ledger. each prioritizer will prioritzie a single tx
+    #
+    # NOTE we create TX_SET_SIZE - 1 prioritizers because there is at always
+    # one reserved tx for unprioritized
+    #
+    # TODO is this even necessary? can't we just add addresses which belong to
+    # uncreated accounts and be done with it?
+    prioritizer_accounts_num = TX_SET_SIZE - 1
```

Comment:
I don't think prioritizers need to be created :+1:

Reply:
they don't in general, but in here they are the payment destination addresses so i must created them

Reply:
i can bypass this easily tho